utils/python/cime_util.py

```diff
@@ -93,7 +58,7 @@ def run_cmd(cmd, ok_to_fail=False, input_str=None, from_dir=None, verbose=None,
     >>> run_cmd('ls file_i_hope_doesnt_exist')
     Traceback (most recent call last):
         ...
```

Comment:
This may break stuff. At one point, the perl/csh was evaluating the success of a command by grepping the output for FAIL instead of checking return code. I added "FAIL" here instead of trying to track down and fix all the places where that was happening.